engine/baml-lib/baml-core/src/lib.rs

```diff
@@ -268,6 +171,152 @@ fn validate_type_builder_blocks(
     }
 }
 
+pub fn run_validation_pipeline_on_db(
+    db: &mut internal_baml_parser_database::ParserDatabase,
+    diagnostics: &mut Diagnostics,
+) {
+    validate::validate(db, Default::default(), diagnostics);
```

Comment:
nit: Using the default method on the type's name is microscopically easier to read than the generic `Default::default()`. `validate::validate(db, SchemaAst::default(), diagnostics);` > `validate::validate(db, Default::default(), diagnostics)`.